Address a couple lingering issues higlighted by Lim (#94)
MobleyLab/basic_simulation_training: paper/basic_training.tex

This neglect of especially short range nonbonded interactions between near neighbors is called ``exclusion'', and energy functions typically specify which interactions are excluded.

The transition to torsions, especially proper torsions, is where exclusions typically end.
However, many all-atom energy functions commonly used in biomolecular simulations retain only \emph{partial} nonbonded interactions between terminal atoms involved in a torsion.
The atoms involved in a torsion, if numbered beginning with 1, would be 1, 2, 3, and 4, so the terminal atoms could be called atoms 1 and 4, and nonbonded interactions between such atoms are called ``1-4 interactions''.
These interactions are often present but reduced, though the exact amount of reduction differs by the energy function or force field family.
For example, the AMBER family force fields usually reduce 1-4 electrostatics to $\frac{1}{1.2}$ of their original value, and 1-4 Lennard-Jones interactions to $\frac{1}{2}$ of their original value.
1-4 interactions are essentially considered the borderline between the bonded and non-bonded regions.

> Because these are short-range interactions, they are potentially quite strong and there is potentially a risk of them overwhelming longer-range interactions, hence their typical reduction.

Reviewer: The use of `potentially` twice so close together sounds a bit awkward.
```suggestion
These short-range interactions can be quite strong and there is potentially a risk of them overwhelming longer-range interactions, hence their typical reduction.
```

Author: Good idea, I'll get that.

\subsubsection{Motivation}

The calculation of non-bonded interactions is generally the most time-consuming step of classical energy calculation.
While the number of type of bonded interactions remain unchanged during an MD simulation, the strength and importance of non-bonded interactions varies substantially as a simulation proceeds.

Additionally, Coulombic interactions fall off only very slowly with distance, as $r^{-1} $, further complicating handling of non-bonded interactions in two different ways.

> First, calculating all Coulomb interactions over a periodic system results in needing to compute a sum which is conditionally convergent --- that is, the value of the sum depends on the order in which it is evaluated~\cite{LeachBook}, meaning we must exercise extreme care or the result will be ambiguous.

Reviewer: ```suggestion
First, calculating all Coulomb interactions over a periodic system results in needing to compute a sum which is conditionally convergent --- that is, the value of the sum depends on the order \emph{in which it is evaluated}~\cite{LeachBook}, meaning we must exercise extreme care or the result will be ambiguous.
```

Seems like a pretty important point to make. Might be worthwhile to italicize it.

Author: OK, I'll fix in my other PR (Heather merged this one).
